ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -39,6 +42,8 @@ class DrawerDemoController: DemoController {
         container.addArrangedSubview(createButton(title: "Show always as slideover, resizable", action: #selector(showBottomDrawerCustomContentControllerButtonTapped)))
 
         container.addArrangedSubview(createButton(title: "Show with focusable content", action: #selector(showBottomDrawerFocusableContentButtonTapped)))
+        
```

Comment:
do you have swift lint installed? it seems like there should be syntax warnings in both of your files

Reply:
Installed swift lint and fixed all the warnings, thanks for calling out!